wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -935,7 +932,9 @@ extension ZMUserSession: ZMSyncStateDelegate {
 
         managedObjectContext.performGroupedBlock { [weak self] in
             self?.notifyThirdPartyServices()
+            self?.observeMLSGroupVerificationStatus()
```

Comment:
Doesn't the same thing also apply to verification status? What if the epoch changes during the quick sync? Wouldn't we miss the epoch change and therefore not update the verifications status?

Reply:
Moved it to init() based on our discussion in the office.